k8s/namespaces/unspec/ithc.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: unspec-service
+spec:
+  values:
+    java:
+      ingressHost: unspec-service-ithc.service.core-compute-ithc.internal
```

Comment:
Its already in https://github.com/hmcts/hmcts-charts/blob/master/stable/unspec-service/values.yaml#L17-L18
```suggestion
```

Reply:
done